operations/helm/charts/mimir-distributed/templates/query-scheduler/query-scheduler-pdb.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.query_scheduler.podDisruptionBudget -}}
+apiVersion: {{ include "mimir.podDisruptionBudget.apiVersion" . }}
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "mimir.resourceName" (dict "ctx" . "component" "query-scheduler") }}
+  labels:
+    {{- include "mimir.labels" (dict "ctx" . "component" "query-scheduler") | nindent 4 }}
```

Comment:
can you add namespace here too?

Reply:
we've merged a PR where this whole file can be simply generated with a named template, see https://github.com/grafana/mimir/pull/2393